indico/htdocs/js/indico/modules/designer.js

```diff
@@ -104,13 +104,20 @@
             return html;
         }).bind(item);
 
-        items[item.id] = item;
         return item;
     }
 
-    function createItemFromObject(obj) {
+    function createItemFromObject(obj, isBackside) {
         delete obj.id;
-        return _.extend(createItem(), obj);
+        var newItem = createItem();
```

Comment:
just a matter of personal preference but IMO this is clearer instead of assigning a new item, just to replace it with another new item the line after

    var newItem = _.extend(createItem(), obj);